scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -359,6 +359,9 @@ def add_parse_arguments(parser = None):
     parser.add_argument("--enable-size-report", action="store_true",
                         help="Enable expensive computation of RAM/ROM segment sizes.")
 
+    parser.add_argument("--create-rom-ram-report", action="store_true",
+                        help="Enable generate rom.json and ram.json for all projects.")
+
```

Comment:
I'd suggest a bit more explicit description. E.g.
```
"Add `--target footprint` argument to cmake build call, generating detailed ram/rom json reports for each build.
```

Reply:
Thanks, rewrote it now based on your suggestion, please take a look